sdk/cosmos/azure-cosmos/azure/cosmos/_cosmos_client_connection.py

```diff
@@ -1287,12 +1301,13 @@ def CreateItem(
 
         if base.IsItemContainerLink(database_or_container_link):
             options = self._AddPartitionKey(database_or_container_link, document, options)
-        return self.Create(document, path, "docs", collection_id, None, options, **kwargs)
+        return self.Create(document, path, "docs", collection_id, None, options, request_context, **kwargs)
 
     def UpsertItem(
         self,
         database_or_container_link: str,
         document: Dict[str, Any],
+        request_context: Dict[str, Any],
```

Comment:
Adding a required positional parameter here is breaking - it must be added as the last parameter, and must be `Optional` with a default of `None`.

Reply:
This also applies to other methods below.

Reply:
Removed request context as not needed